app/models/shipit/deploy_spec/lerna_discovery.rb

```diff
@@ -41,6 +53,11 @@ def lerna_json
         file('lerna.json')
       end
 
+      def lerna_lerna
+        lerna_config = lerna_json.read
+        Gem::Version.new(JSON.parse(lerna_config)['lerna'])
```

Comment:
Should/could we memoize this? Could this JS file every be large / expensive to parse?

Reply:
fixed!